views/pages/DashboardPage/DashboardPage.test.jsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react'
+import { render, screen, within, waitFor } from '@testing-library/react'
+import { AuthContext, NotificationContext } from '../../contexts'
+import { MemoryRouter } from 'react-router-dom'
```

Comment:
what do you think about organizing these by external/internal import?

```suggestion
import { render, screen, within, waitFor } from '@testing-library/react'
import { MemoryRouter } from 'react-router-dom'
import { AuthContext, NotificationContext } from '../../contexts'
```

Reply:
529a6b0